crates/node/src/cli.rs

```diff
@@ -36,17 +36,41 @@ pub struct ExtendedRunCmd {
     #[clap(flatten)]
     pub run_cmd: RunCmd,
 
+    /// Load a custom chain-spec from an url
     #[clap(long)]
-    pub testnet: Option<Testnet>,
+    pub chain_spec_url: Option<String>,
+
+    /// Load a custom index.json file for downloading custom assets
+    /// Check documentation in
+    /// https://github.com/keep-starknet-strange/madara/blob/main/docs/configs.md
+    #[clap(long)]
+    pub configs_url: Option<String>,
+
+    /// Disable madara default configs:
+    /// - Fetching index.json, genesis.json and genesis assets
+    /// - Fetching default chain specs
+    #[clap(long)]
+    pub disable_madara_configs: bool,
+
+    /// Disable automatic url fetching for madara config files
+    #[clap(long)]
+    pub disable_url_fetch: bool,
```

Comment:
Hey, I have been thinking and I would rather prefer to disable fetch URL being the default.
By default do nothing. It seems more logic.

And then you can pass `--fetch-chain-specs` or  `--fetch-chain-specs=<your_url>`.
The first one will use our default value, the second the one specified by the user.
If you don't pass it, it will be set to None.

To do so use https://docs.rs/clap/latest/clap/struct.Arg.html#method.default_missing_value
```rust
 #[arg(long, num_args = 0..=1, default_missing_value("my_default_url"))]
pub fetch-chain-specs: Option<String>,
```

Reply:
Same for disable_madara_config.